registry/garbagecollect.go

```diff
@@ -107,9 +115,14 @@ func markAndSweep(ctx context.Context, storageDriver driver.StorageDriver) error
 		return nil
 	})
 
+	emit("\n%d blobs marked, %d blobs eligible for deletion", len(markSet), len(deleteSet))
 	// Construct vacuum
 	vacuum := storage.NewVacuum(ctx, storageDriver)
 	for dgst := range deleteSet {
+		if dryRun {
+			emit("deleting %s", dgst)
```

Comment:
I think this should be moved after the `if` statement, so that the information is still printed outside dry-run mode, and this `emit` should be replaced with something like "eligible for deletion: %s".